dropwizard-app/src/test/java/org/coner/util/ApiEntityUtils.java

```diff
@@ -0,0 +1,53 @@
+package org.coner.util;
+
+import org.coner.api.entity.Event;
+import org.coner.api.entity.Registration;
+
+import java.util.Date;
+
+/**
+ *
+ */
+public class ApiEntityUtils {
+
+    protected ApiEntityUtils() {
+        // prevents calls from subclass
+        throw new UnsupportedOperationException();
+    }
```

Comment:
How about this:
```java
public final class SomeUtilityClass {
    private SomeUtilityClass() {
        // no-op
    }
}
```
I think that should satisfy the checkstyle recommendation and still let the compiler and IDE send the right messages to the developer